### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_Select Source-Target_SchemaCompare_Screenreader: Incorrect total count announced by nvda

### ISSUE THREAD

**Issue by ShwetaBaglane** [microsoft/azuredatastudio#6944]
[“Check out Accessibility Insights! !](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
Application Name: Azure Data Studio
Application Version: 1.10.0-insider (user setup)
Commit: 1ff9a2ec4c7b88be01666bdd470bbbb4ab95716e
Date: 2019-08-09T07:49:00.254Z
VS Code 1.37.0
Electron: 4.2.7
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
Nvda Version: 2019.2rc2
Jaws Version:  2019.1903.47 ILM
Serial Number: 32810

**Additional Details:**
Tools Name: NVDA, JAWS
MAS References: MAS1.3.1, MAS4.2.3

**Repro Steps:**

1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Schema Compare extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Schema compare from Context menu.
8. Navigate till Select Source or Select Target & hit enter.
9. Start screen reader as NVDA.
10. Navigate to Database Edit using tab key.
11. Navigate to Tree Items using arrow key.

**Actual:**
NVDA announces incorrect count as 6 for Database Edit of Source and Target.

**Same issue is observed with JAWS**

**Expected:**
NVDA should announce correct total TreeItems count. In this case, total count is 13.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: (https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
Screenreader users will not get correct information about total available options/ items present in Database edit.

**MAS Reference:**
MAS 1.3.1
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
MAS 4.2.3
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={30e4fb40-dcc1-45b4-b714-5e2c28b23ae8}

**Attachment for Reference:**
NVDA Speech Viewer:
![image](https://user-images.githubusercontent.com/53556874/63669308-429dd400-c7f7-11e9-97ba-8f0f72ff97c5.png)

Jaws Speech Viewer:
![image](https://user-images.githubusercontent.com/53556874/63669416-9d373000-c7f7-11e9-8b13-d4c1e78cd593.png)

**Comment by Benjin** [comment]
Looks like an `aria-rowcount` property will need to be added to VSCode's ITree component.  Pinged @anthonydresser to confirm

**Comment by Benjin** [comment]
From Anthony, they weren't able to make that component meet accessibility criteria, so it gets replaced with a standard dropdown in accessibility mode.  Recommended to close.

**Comment by Benjin** [comment]
@fsteffi The issue/fix notes say that the solution from upstream is to make the fix present when running in accessibility mode.  Did you confirm that this issue persists when running ADS in accessibility mode?

![image](https://user-images.githubusercontent.com/1609827/66663936-7f375a80-ec00-11e9-91ea-b40fb2b08bd8.png)

**Comment by fsteffi** [comment]
Checked and verified in below version the bug is no longer reproduced and hence closing the bug.

Version Details:
Version: 1.12.0-insider
Commit: 8e1a2248e47925eb825b0ce037b96b6c87798a8a
Date: 2019-10-04T07:52:03.973Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

![#6944A11y_AzureDataStudio(Schema,Dacpac,Backup)_Select Source-Target_SchemaCompare_Screenreader](https://user-images.githubusercontent.com/24580570/66801673-7d8ad100-ef37-11e9-9df9-e470b7ff0f2c.png)
